### PR TITLE
Replace pre-commit action

### PR DESCRIPTION
**Pull request by kse3hi**

### CODE REVIEW

**Review by dennismeister93 — APPROVED** [review]
tests are not succesful

**Review by dennismeister93 — CHANGES_REQUESTED** [review]
tests are not working

File: sdk-installer/src/run.py [hunk 1]
```diff
@@ -51,14 +51,14 @@ def get_required_sdk_version_python() -> str:
...
-    sdk_version: str = "0.12.0"
+    sdk_version: str = "0.13.0"
     requirements_path = os.path.join(
         get_workspace_dir(), "app", "requirements-velocitas.txt"
     )
     if os.path.exists(requirements_path):
         with open(requirements_path, encoding="utf-8") as requirements_file:
             for line in requirements_file:
-                if line.startswith("velocitas-sdk"):
+                if line.startswith("velocitas_sdk"):
```
Comment (doosuu) [inline sdk-installer/src/run.py:61]: I thought we wanted to accept both `velocitas-sdk` and `velocitas_sdk` here?

Comment (kse3hi) [reply sdk-installer/src/run.py:61]: I go a little bit deeper, and figure out that PEP503 is obligate to normalize names of packages and replace all `-`,`.` and `_` with `-`.
LInk: https://peps.python.org/pep-0503/#normalized-names

As far as it is required renaming and releasing new package, and can lead for inconsistency, I've decided to separate this activity with the new task.

**Review by doosuu — APPROVED** [review]
lgtm

**Comment by kse3hi** [comment]
> tests are not working

Fixed

**Review by dennismeister93 — APPROVED** [review]
lgtm

**Pull request merged**